StudentDriver/StudentDriver/Models/User.cs

```diff
@@ -12,7 +12,7 @@ public class User
         public string Email { get; set; }
         public string FirstName { get; set; }
         public string LastName { get; set; }
-        public int DrivingSchoolId { get; set; }
+        public string DrivingSchoolId { get; set; }
```

Comment:
Uh-oh, looks like your id is a string. But the server's id is a Number: 
````javascript 
schoolId: Number
````

What should we conform too? @DylanWalseth @dobber1611